src/ray/object_manager/plasma/create_request_queue.h

```diff
@@ -151,7 +151,8 @@ class CreateRequestQueue {
   /// Process a single request. Sets the request's error result to the error
   /// returned by the request handler inside. Returns OK if the request can be
   /// finished.
-  Status ProcessRequest(std::unique_ptr<CreateRequest> &request, bool fallback_allocator);
+  Status ProcessRequest(bool fallback_allocator, std::unique_ptr<CreateRequest> &request,
```

Comment:
Supernit: I'm mostly just curious, since the `request` seems like the most "important" argument while `fallback_allocator` tweaks the semantics of how the request should be processed, was there a particular reason that you moved `request` to be the second argument? We've typically tried to order arguments by how important/central they are to the function, and have tried to keep these kind of booleans that tweak the function's semantics towards the end of the argument ordering, but this hasn't been a hard style rule.

Reply:
Yea I was following the Google style guide where in-parameters come before out-parameters: https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

If we have other conventions I can change this.

Reply:
Ah, that makes sense! Thanks for pointing that out, I momentarily forgot that `request` is an output parameter. 😁